distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc

```diff
@@ -10,15 +10,15 @@ To exit this mode, click the `?` again or press `escape`.
 
 === Accessing ${catalog-ui}
 
-The default location for the ${catalog-ui} is ${secure_url}/search/catalog
+The default location for ${catalog-ui} is ${secure_url}/search/catalog
 
 .Catalog UI Guest Users
```

Comment:
What does this line do? I see
![image](https://cloud.githubusercontent.com/assets/8041246/25686048/4e4fb5a0-3020-11e7-9a5e-010f38f2d2a9.png)